.devcontainer/images/.claude/commands/do/questions.md

```
@@ -0,0 +1,151 @@
+    action: "Detect the tech stack"
+    checks:
+      - "package.json → Node.js/npm"
+      - "go.mod → Go"
+      - "Cargo.toml → Rust"
+      - "pyproject.toml → Python"
+    output: "test_command, lint_command, build_command"
+```
+
+**Output Phase 4.0:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)